Analysis/Tasks/jetfinder.cxx

```diff
@@ -64,11 +64,9 @@ struct JetFinderTask {
   fastjet::AreaDefinition areaDefBkg{areaType, ghostSpec};
   fastjet::Selector selBkg = fastjet::SelectorAbsRapMax(rapBkg);
 
-  // TODO: use abs, eventually use pt
   // TODO: use values from configurables
   // TODO: add eta cuts
-  Filter trackCuts = (aod::track::signed1Pt < 10.f) &&
-                     (aod::track::signed1Pt > -10.f);
+  Filter trackCuts = nabs(aod::track::pt) > 0.1f;
 
```

Comment:
Shouldn't pt be always positive anyway?

Reply:
You are right, I'll fix it as soon as tests finish.

Reply:
Done.